wgengine/router/router_func.go

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2020 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package router
+
+import (
+	"tailscale.com/types/logger"
+)
+
+// RoutingFuncs are functions implementing a routing configuration interface.
```

Comment:
now that you have this, this can just implement Router, rather than having funcRouter and NewFuncRouter (which just discards its logger.Logf anyway).

You'd just need to rename these struct fields to UpFunc, DownFunc, SetFunc to free up the names for the methods.

I'd call it "FuncsRouter" instead of "Routing" probably, to match "Router"